homeassistant/components/homeassistant/triggers/time.py

```diff
@@ -202,6 +217,15 @@ def update_entity_trigger(
     to_track: list[TrackEntity] = []
 
     for at_time in config[CONF_AT]:
+        if isinstance(at_time, template.Template):
+            render = template.render_complex(at_time, variables, limited=True)
+            try:
+                at_time = _TIME_AT_SCHEMA(render)
+            except vol.Invalid as exc:
+                raise HomeAssistantError(
+                    "Expected HH:MM, HH:MM:SS or Entity ID with domain 'input_datetime' or 'sensor'"
+                ) from exc
```

Comment:
The full error message from the automation is something like:
> Got error 'Expected HH:MM, HH:MM:SS or Entity ID with domain 'input_datetime' or 'sensor'' when setting up triggers for automation

I think the exception text needs to mention it's a time trigger's "at" template that rendered to a value we could not understand?